ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -324,6 +324,34 @@ spec:
         - name: dest-registry-credentials
           workspace: registry-push-credentials
 
+    - name: set-cert-project-repository
+      runAfter:
+        - copy-bundle-image-to-released-registry
+      taskRef:
+        name: set-cert-project-repository
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: cert_project_id
+          value: "$(tasks.certification-project-check.results.certification_project_id)"
+        - name: registry
+          value: "$(params.dest_registry)"
+        - name: repository
+          value: "$(params.dest_image_namespace)/$(tasks.certification-project-check.results.certification_project_id)"
+        - name: docker_config_file_name
+          value: ".dockerconfigjson"
+        - name: pyxis_url
+          value: "$(tasks.set-env.results.pyxis_url)"
+        - name: pyxis_ssl_secret_name
+          value: "$(params.pyxis_ssl_secret_name)"
+        - name: pyxis_ssl_cert_secret_key
+          value: "$(params.pyxis_ssl_cert_secret_key)"
+        - name: pyxis_ssl_key_secret_key
+          value: "$(params.pyxis_ssl_key_secret_key)"
+      workspaces:
+        - name: registry-credentials
+          workspace: registry-push-credentials
```

Comment:
Ideally, we should not use credentials which have permission to push to the registry because that violates the principle of least privilege. The credentials stored on the project _should_ only be used to pull its images. @Allda is there a more appropriate service account with read-only access we can use instead?

Reply:
That's actually a really good point. We don't have a token with just read access. I'll create one today and share it with @mporrato